e2e/main.spec.ts

```diff
@@ -0,0 +1,25 @@
+import {SpectronClient} from 'spectron';
+
+import commonSetup from './common-setup';
+
+describe('CATcher App', function () {
+  commonSetup.apply(this);
+
+  let browser: any;
+  let client: SpectronClient;
+
+  beforeEach(function () {
+    client = this.app.client;
+    browser = client as any;
```

Comment:
A clarification: is the `browser` variable used because we need to broaden the `client`'s type into `any` - in order to use some of the WebdriverIO methods? (such as `getText` ?)

Reply:
Yeap!